allensdk/internal/api/mtrain_api.py

```diff
@@ -14,7 +14,7 @@ class MtrainApi(PostgresQueryMixin):
 
     def __init__(self, api_base='http://mtrain:5000'):
         self.api_base = api_base
-        self.behavior_lims_api = BehaviorLimsApi()
+        self.behavior_lims_api = BehaviorLimsApi
```

Comment:
is there any need for this to be an instance attribute at all? The only references I see to `self.behavior_lims_api` are calls to `behavior_session_id_to_foraging_id`

Reply:
Good point. As discussed in person, changed the code to use the static methods directly.